backpack/extensions/firstorder/base.py

```diff
@@ -1,6 +1,14 @@
+from backpack.extensions.backprop_extension import BackpropExtension
 from backpack.extensions.module_extension import ModuleExtension
 
 
 class FirstOrderModuleExtension(ModuleExtension):
     def backpropagate(self, ext, module, g_inp, g_out, bpQuantities):
         return None
```

Comment:
Add file to fully-documented. Replace `return None` by `pass` (cleaner for 'does nothing')